RELEASE-NOTES.md

```diff
@@ -3,9 +3,9 @@ id: Release notes
 section: get-started
 releaseNoteTOC: true
 ---
-## 2020.12 release notes (2020-09-16)
+## 2020.12 release notes (2020-09-17)
 Packages released:
-- [@patternfly/patternfly@v4.41.1](https://www.npmjs.com/package/@patternfly/patternfly/v/4.41.1)
+- [@patternfly/patternfly@v4.42.2](https://www.npmjs.com/package/@patternfly/patternfly/v/4.42.2)
```

Comment:
won't this be 4.42.1? It's 4.42.0 currently - https://github.com/patternfly/patternfly/releases/latest

Reply:
It'll be 4.42.2 after your  release notes PR #3476 releases in a few minutes